cluster/ci/config/prow/plugins.yaml

```diff
@@ -294,7 +294,9 @@ approve:
 - repos:
   - openshift/machine-api-operator
   - openshift/cluster-api-provider-aws
+  - openshift/cluster-api-provider-libvirt
   implicit_self_approve: false
+  lgtm_acts_as_approve: false
```

Comment:
Err -- if you need a novel set of options, add a new entry to the `approve` list, otherwise you're changing this option for the other repos in the list as well

Reply:
That's the idea, I need this to be changed for all 3 repos.

Reply:
Of course. D'oh